homeassistant/components/climate/melissa.py

```diff
@@ -123,6 +124,15 @@ def state(self):
             return self.melissa_state_to_hass(
                 self._cur_settings[self._api.STATE])
 
+    @property
+    def state_attributes(self):
+        """Add the attributes that's available."""
+        data = super(MelissaClimate, self).state_attributes
```

Comment:
Platforms are not allowed to implement `state_attributes`, they should implement `device_state_attributes`